pkgs/applications/misc/plover/default.nix

```diff
@@ -1,37 +1,138 @@
-{ lib, fetchFromGitHub, python3Packages, wmctrl, qtbase, mkDerivationWith }:
+{
+  lib,
+  fetchFromGitHub,
+  python310Packages,
```

Comment:
Can we have an explanation to why 3.10 specifically?

Reply:
I've added a comment for that